higher_health/views.py

```diff
@@ -57,6 +57,17 @@ def get_initial(self):
                 initial_data[
                     "history_pre_existing_condition"
                 ] = triage.preexisting_condition
+
+                initial_data["symptoms_fever"] = triage.fever
+                initial_data["symptoms_cough"] = triage.cough
+                initial_data["symptoms_sore_throat"] = triage.sore_throat
+                initial_data[
+                    "symptoms_difficulty_breathing"
+                ] = triage.difficulty_breathing
+                initial_data["symptoms_muscles_hurt"] = triage.muscle_pain
+                initial_data["symptoms_taste"] = "yes" if triage.taste else "no"
+                initial_data["medical_exposure"] = triage.exposure
```

Comment:
This is the only one that should be set. We want people to always complete the symptoms questions afresh. 

Unless the ticket had a different request? (let's chat in slack)